NextDom/nextdom-core
# update disposition action in scenario prog

to test : 
- open some scenario and check disposition
ca rend les scenario editable sur ecran pas large

before: 
![image](https://user-images.githubusercontent.com/18350629/48313506-e9ddb180-e5bd-11e8-88f4-ad93a442ec79.png)

after:
![image](https://user-images.githubusercontent.com/18350629/48313528-18f42300-e5be-11e8-853b-2cddae30c33a.png)
